src/main/java/org/terasology/metalrenegades/ai/component/CitizenComponent.java

```diff
@@ -20,6 +20,6 @@
 /**
  * A component added to prefabs containing spawnable AI agents.
  */
-public class ResidentComponent implements Component {
```

Comment:
If you are going to rename the class can you also rename the file.

Reply:
I think the file has been renamed? It may not be showing up on github properly...